tests/slsa_analyzer/provenance/expectations/cue/test_cue_validator.py

```diff
@@ -12,16 +12,17 @@
 from macaron.database.table_definitions import CUEExpectation
 from macaron.slsa_analyzer.provenance.expectations.cue.cue_validator import get_target, validate_expectation
 
-EXPEC_RESOURCE_PATH = Path(__file__).parent.joinpath("resources")
+EXPECT_RESOURCE_PATH = Path(__file__).parent.joinpath("resources")
 PROV_RESOURCE_PATH = Path(__file__).parent.parent.parent.joinpath("resources")
+PACKAGE_URLLIB3 = "pkg:github.com/urllib3/urllib3"
```

Comment:
some linter objected to the natural change which apparently caused the line to exceed a limit. This avoids that.